src/blockchains/construct_worker.ts

```diff
@@ -22,8 +21,6 @@ export function constructWorker(blockchain: string, settings: any): BaseWorker {
       return new ETHBlocksWorker(settings);
     case 'matic':
       return new MaticWorker(settings);
-    case 'receipts':
-      return new ReceiptsWorker(settings);
```

Comment:
Separate worker is no longer needed. We would use the `ETHWorker` to extract receipts, we can do it like this:

> KAFKA_TOPIC='native_token_transfers:eth_transfers_test_topic,receipts:receipts_test_topic'

If we want to extract transfers and receipts at once. If for some reason we want to have only receipts deploy, we can do:

> KAFKA_TOPIC='receipts:receipts_test_topic'